src/collective/collectionfilter/upgrades.zcml

```diff
@@ -100,5 +100,13 @@
       handler=".upgrades.upgrade_to_plone6"
       />
 
+  <genericsetup:upgradeStep
+     title="collective.collectionfilter: Upgrade v12 to v13"
+     description="upgrade to version 13"
+     source="12"
+     destination="13"
+     profile="collective.collectionfilter:upgrade-13"
+     handler=".upgrades.upgrade_registry_13"
+     />
 
```

Comment:
```suggestion
  <genericsetup:upgradeDepends
     title="collective.collectionfilter: Upgrade v12 to v13"
     description="upgrade to version 13"
     source="12"
     destination="13"
     profile="collective.collectionfilter:default"
     import_profile="collective.collectionfilter:upgrade-13"
     />

```

And the registration of the upgrade profile `collective.collectionfilter:upgrade-13` is missing in `configure.zcml`